src/extension/module.js

```diff
@@ -2337,7 +2342,29 @@ import sampleRUM from './rum.js';
             custom: true,
             id: id || `custom-plugin-${i}`,
             condition,
-            button: {
+            container: containerId,
+          };
+
+          if (isBadge) {
+            plugin.feature = true;
+            plugin.elements = [{
+              tag: 'span',
+              text: title,
+              attrs: {
+                class: 'hlx-sk-badge',
+              },
+            }];
+            // set badge styles
+            const isDarkSchemePreferred = () => window?.matchMedia?.('(prefers-color-scheme:dark)')?.matches ?? false;
+            if (isDarkSchemePreferred()) {
+              if (badgeTextColorDark) sk.style.setProperty('--hlx-sk-badge-color', badgeTextColorDark);
```

Comment:
I think this would redefine the badge colors for all badges, and the last one wins...